From a330b289b4f9cbc708df4262dbe2f4857e65c300 Mon Sep 17 00:00:00 2001
From: Omar Sandoval <osandov@osandov.com>
Date: Thu, 21 Apr 2016 22:02:38 -0700
Subject: [PATCH] Use a GTK_WINDOW_POPUP for notification window

This prevents the notification from taking the focus and getting stuck
on a tag in dwm.
---
 xfce4-notifyd/xfce-notify-window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xfce4-notifyd/xfce-notify-window.c b/xfce4-notifyd/xfce-notify-window.c
index 285a70e3efe6..e4353edc5097 100644
--- a/xfce4-notifyd/xfce-notify-window.c
+++ b/xfce4-notifyd/xfce-notify-window.c
@@ -736,7 +736,7 @@ xfce_notify_window_new_with_actions(const gchar *summary,
     XfceNotifyWindow *window;
 
     window = g_object_new(XFCE_TYPE_NOTIFY_WINDOW,
-                          "type", GTK_WINDOW_TOPLEVEL, NULL);
+                          "type", GTK_WINDOW_POPUP, NULL);
 
     xfce_notify_window_set_summary(window, summary);
     xfce_notify_window_set_body(window, body);
-- 
2.8.0

